source/_integrations/probe_plus.markdown

```diff
@@ -0,0 +1,70 @@
+---
+title: Probe Plus
+description: Instructions on how to integrate Probe Plus food temperature probes into Home Assistant.
+ha_release: 2025.5.0
+ha_category:
+  - Sensor
+ha_iot_class: local_push
+ha_config_flow: true
+ha_domain: probe_plus
+ha_platforms:
+  - sensor
+ha_bluetooth: true
+ha_codeowners:
+  - '@pantherale0'
+ha_integration_type: device
+ha_quality_scale: bronze
+---
+
+The **Probe Plus** {% term integration %} allows you to view food temperature probe details supported by [Probe Plus](https://play.google.com/store/apps/details?id=com.yscoco.thermoex) through Home Assistant.
+
```

Comment:
is there a prerequisite? For example, do I need to have an app installed on my phone, or will it connect to Home Assistant directly?

Reply:
No, the app is only there as a reference to the devices it supports "in theory".

The BLE logic and parsers are reverse engineered from the app itself (which are pretty generic anyway).. In theory this would support many different devices although I only have 1 to test with.